components/navbar/Navbar.js

```diff
@@ -145,16 +145,19 @@ export default function Navbar() {
                   />
                 </Link>
               </div>
-              <div className="hidden md:block">
+             <div className="hidden md:block">
```

Comment:
I think this indentation change is not correct

Reply:
The prettier config was surprisingly not working yesterday but stated working now. I think the fix is okay.